src/plots/cartesian/position_defaults.js

```diff
@@ -36,16 +39,26 @@ module.exports = function handlePositionDefaults(containerIn, containerOut, coer
         }
     }, 'anchor');
 
-    if(anchor === 'free') coerce('position', dfltPosition);
-
-    Lib.coerce(containerIn, containerOut, {
+    var side = Lib.coerce(containerIn, containerOut, {
         side: {
             valType: 'enumerated',
             values: letter === 'x' ? ['bottom', 'top'] : ['left', 'right'],
             dflt: dfltSide
         }
     }, 'side');
 
+    if(anchor === 'free') {
```

Comment:
In addition to ensuring the new attributes aren't included in the x schema (that @archmoj is going to make a commit for?), I guess this whole block only applies to y, with the exception of `position` that still needs to be coerced for free x axes?

Reply:
Fair point. Have added in some more logic there.